### DISCUSSION TITLE
Hello i set a one mojo fee on my last transaction and they charge me one complete chia

### DISCUSSION THREAD

**Discussion by heshelk2** [Chia-Network/chia-blockchain#9015] [Support]
Hello, today i make a transaction to mojodice for .4 chia, and set a fee of one mojo, but the transaction charge me one complete chia please help! my address is xch1ccsucg0umarzj0cxst8dh8qr223h929e0jz5mm2ekcmzkmnscxhq46qxjw

**Comment by Tomaszu97** [comment]
no it did not, as seen here : [link](https://xchscan.com/address/xch1ccsucg0umarzj0cxst8dh8qr223h929e0jz5mm2ekcmzkmnscxhq46qxjw)

**Comment by OldButtKicking** [comment]
unfortunately you are not going to like the answer. The fee is in chia not mojo you would have to enter a trillionth of a chia for a mojo or better yet 0 you do not have to pay a fee. I'm afraid the xch is lost.
![image](https://user-images.githubusercontent.com/58090640/143189878-27871444-c8af-4dba-a69f-1c0200553ec5.png)

If it is any consolation I made an even worse mistake I managed to send a xch to a flx account last I saw of that :(
